### PR TITLE
Issue 695

### PR DESCRIPTION
**Pull request by KorynLA**
New updates with the proposed changes from PR #909 :
- Used '[]' as wrappers for backend error case numbers.
- Updated frontend to use match and append hyperlinked case numbers
- Updated backend test_crawler_expunger to no longer fail with newly wrapped error case numbers.

### CODE REVIEW

File: src/backend/expungeservice/expunger.py [hunk 1]
```diff
@@ -213,7 +213,7 @@ def _filter_cases_with_errors(charges: List[Charge]):
         cases_with_unrecognized_disposition: Set[Tuple[str, str]] = set()
         for charge in charges:
             if charge.blocks_other_charges():
-                case_number = charge.case()().case_number
+                case_number = f"[{charge.case()().case_number}]";
```
Comment (KentShikama) [inline src/backend/expungeservice/expunger.py]: semicolon

File: src/frontend/src/components/SearchResults/index.tsx [hunk 2]
```diff
@@ -12,8 +12,19 @@ export default class SearchResults extends React.Component<Props> {
     const errors = ( this.props.record.errors ?
       this.props.record.errors.map(((errorMessage: string, errorIndex: number) => {
         const id= "record_error_" + errorIndex;
+        
+        const errorMessageArray = errorMessage.split(/(\[.*?\])/g); 
+        console.log("Message" + errorMessageArray);
```
Comment (KentShikama) [inline src/frontend/src/components/SearchResults/index.tsx]: log message

File: src/backend/expungeservice/expunger.py [hunk 3]
```diff
@@ -24,7 +24,7 @@ def run(self) -> bool:
         """
         open_cases = [case for case in self.record.cases if not case.closed()]
         if len(open_cases) > 0:
-            case_numbers = ",".join([case.case_number for case in open_cases])
+            case_numbers = ",".join([f"[{case.case_number}]" for case in open_cases])
```
Comment (wittejm) [inline src/backend/expungeservice/expunger.py]: While we're here you can fix this space:
```suggestion
            case_numbers = ", ".join([f"[{case.case_number}]" for case in open_cases])
```

**Comment by wittejm** [comment]
merged this via an updated branch #920

**Pull request closed without merging**